main.py

```diff
@@ -1,100 +1,41 @@
-
-    filters = (
-        landsat.filterBounds(roi).filterDate('2020-01-01', '2020-12-31').filter(ee.Filter.lt('CLOUD_COVER', 40))
-        .map(lambda img: img.multiply(ee.Number(img.get("TEMPERATURE_MULT_BAND_ST_B10")))  # Gain
-            .add(ee.Number(img.get("TEMPERATURE_ADD_BAND_ST_B10")))  # Offset
-            .copyProperties(img, img.propertyNames())
-        )
-    )
-
-    num_images = filters.size().getInfo()
-    print(f'Number of images after filtering: {num_images}')
+    city_coordinates = [location.latitude, location.longitude]
+    print(f"The center coordinates for your city are {city_coordinates}")
+    return city_coordinates
 
-    thermal_infra_image = landsat.median()
-    Map.addLayer(thermal_infra_image.clip(roi),{},'tir_median', False)
-
-    # Calculate the mean temprature value over a region (roi), setting the spatial resolution in meters (scale=100)
-    tir_mean = ee.Number(thermal_infra_image.reduceRegion(
-        reducer= ee.Reducer.mean(), geometry=roi, scale=100
-    ).values().get(0))
-
-    urban_head_island = thermal_infra_image.expression(
-        '(tir - mean)/mean', 
-        {
-            'tir':thermal_infra_image,
-            'mean':tir_mean
-        }).rename('urban_heat_is')
-    Map.addLayer(urban_head_island.clip(roi),{},'urban_heat_is',False)
-    
-    # Classifing: each pixel based on the temperature to be set to the corresponding color.
-    uhi_class = (ee.Image.constant(0)
-        .where(urban_head_island.gte(0).And(urban_head_island.lt(0.005)),1)
-        .where(urban_head_island.gte(0.005).And(urban_head_island.lt(0.010)),2)
-        .where(urban_head_island.gte(0.010).And(urban_head_island.lt(0.015)),3)
-        .where(urban_head_island.gte(0.015).And(urban_head_island.lt(0.020)),4)
-        .where(urban_head_island.gte(0.020),5)
-    )
-    Map.addLayer(uhi_class.clip(roi),{
-        "min":1,
-        "max":5,
-        "opacity": 0.50,
-        "palette":["blue","green","yellow","orange","red"]
-    }, "uhi_class", True)
-
-    Map.save("heat_map.html")
-    # Print the number of images in the filtered collection
-    
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Run main program with optional commands.")
-    parser.add_argument("-c","--city", action='store_true', help="Will provide the option to enter a city name.")
+    parser.add_argument("-c", "--city", help="Will provide the option to enter a city name in quotes, ex: 'New York'.")
+    parser.add_argument("-y", "--year", help="Will provide the option to enter to enter a year, ex: 2020.")
     args = parser.parse_args()
-    
+
     if args.city:
-        print("here")
-        location = get_city_coordinates()
+        location = get_city_coordinates(args.city)
     else:
-        #Default the Urban Heat Island Map to Austin, Texas
-        location = [-97.74414483713234,30.26562490046467]
-    
-    main(location)
+        location = Default_Coordinates
```

Comment:
I think you can set the default to `add_argument`? Then you don't need this if/else